Lib/collections/__init__.py

```diff
@@ -46,7 +46,7 @@
     _collections_abc.MutableSequence.register(deque)
 
 try:
-    from _collections import _deque_iterator
+    from _collections import _deque_iterator  # noqa
 except ImportError:
     pass
```

Comment:
`test_collections` passes for me if I remove this import. Does it need to stay? It was added in https://github.com/python/cpython/commit/52f96d3ea39fea4c16e26b7b10bd2db09726bd7c

Reply:
@erlend-aasland @kumaraditya303: Do you know/recall why this symbol is exposed?

Reply:
Oh, `test_deque` fails if the import is removed. I expected `deque` to be tested as part of `test_collections`, but apparently it has its own test. Could you add a comment that it is (apparently) required to expose it in the `collections` module in order for deque iterators to be pickled?

```pytb
======================================================================
ERROR: test_iterator_pickle (test.test_deque.TestBasic.test_iterator_pickle)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/alexw/dev/cpython/Lib/test/test_deque.py", line 640, in test_iterator_pickle
    dump = pickle.dumps((itorg, orig), proto)
_pickle.PicklingError: Can't pickle <class 'collections._deque_iterator'>: attribute lookup _deque_iterator on collections failed

----------------------------------------------------------------------
```

Reply:
Sure, I added a comment.